docs/packet.md

```diff
@@ -40,7 +40,7 @@ Grab the latest version of Terraform and install it.
 ```bash
 echo "https://releases.hashicorp.com/terraform/$(curl -s https://checkpoint-api.hashicorp.com/v1/check/terraform | jq -r -M '.current_version')/terraform_$(curl -s https://checkpoint-api.hashicorp.com/v1/check/terraform | jq -r -M '.current_version')_darwin_amd64.zip"
 sudo yum install unzip
-sudo unzip terraform_0.11.11_linux_amd64.zip -d /usr/local/bin/
+sudo unzip terraform_0.12.6_linux_amd64.zip -d /usr/local/bin/
```

Comment:
It should be or isn't `0.12.12` ?

Reply:
That's right. I've updated that in the latest commit.